env/env.go

```diff
@@ -16,14 +19,42 @@ type Clusters map[string]NodeConfig
 type NodeConfig struct {
 	FileName string
 	Path     string
+	Payload  string
+}
+
+type Cmd interface {
+	FetchSrc() int
+	Show() int
 }
 
 type Info struct {
-	Cluster    Clusters        `mapstructure:"cluster"`
+	SrCluster  Clusters        `mapstructure:"srcluster"`
+	DsCluster  Clusters        `mapstructure:"dscluster"`
```

Comment:
@gildub if you don't expect these to be [de]serialized with [un]marshal() you don't want to mark those with `mapstrcuture:"..."`.